src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -4,11 +4,15 @@
 
 #include <absl/container/flat_hash_map.h>
 #include <absl/strings/str_format.h>
+#include <gmock/gmock-spec-builders.h>
```

Comment:
Have you checked whether this include is needed?

AFAIK Gmock wants users to only include gmock.h and considers all other headers private. gmock.h will include gmock-spec-builders.h transitively.

From gmock.h:

```
This is the main header file a user should include.
```